codersdk/client.go

```diff
@@ -24,6 +24,9 @@ const (
 	SessionCustomHeader = "Coder-Session-Token"
 	OAuth2StateKey      = "oauth_state"
 	OAuth2RedirectKey   = "oauth_redirect"
+
+	// nolint: gosec
+	BypassRatelimitHeader = "X-Coder-Bypass-Ratelimit"
```

Comment:
Why not just allow owner to bypass implicitly without needing to set a header?

Reply:
I thought it may lead to bugs / misunderstandings with other features that rely on the rate limit. For example, a spinning script.

Also, we may not realize if the front end is sending too many requests since we usually test with Owner.